src/main/java/com/restaurant/eatenjoy/util/SHA256Encrypt.java

```diff
@@ -0,0 +1,34 @@
+package com.restaurant.eatenjoy.util;
+
+import java.security.MessageDigest;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class SHA256Encrypt implements EncryptUtil {
+
+	@Override
+	public String encrypt(String msg) {
+
+		try {
+			MessageDigest messageDigest = MessageDigest.getInstance("SHA-256");
+			byte[] hash = messageDigest.digest(msg.getBytes("UTF-8"));
+
+			StringBuffer hexString = new StringBuffer();
+
+			for (int i = 0; i < hash.length; i++) {
+				String hexValue = Integer.toHexString(0xff & hash[i]);
+
+				if (hexValue.length() == 1) {
+					hexString.append("0");
+				}
+
+				hexString.append(hexValue);
+			}
+
+			return hexString.toString();
+		} catch (Exception e) {
+			throw new RuntimeException("null algorithm name " + e);
```

Comment:
cause는 인자로 넘겨줘야 제대로 로그에 찍힙니다~

```suggestion
			throw new RuntimeException("null algorithm name ", e);
```

Reply:
기서 예외가 잡히는 케이스는 `null algorithm name`뿐만이 아닐 것 같습니다~